### PR TITLE
downgrade `importlib-metadata` version to fix taskcluster issue

### PR DESCRIPTION
**Pull request by nnrepos**
due to misuse of deprecated function in `importlib-metadata>=5` inside `flake8`, tests fail here: #534.
downgrading to `importlib-metadata==4.13.0` should fix it.

### CODE REVIEW

**Review by marco-c — CHANGES_REQUESTED** [review]
This should go in test-requirements.txt, since flake8 is not required to run the crawler, it's required just for development

**Comment by nnrepos** [comment]
> This should go in test-requirements.txt, since flake8 is not required to run the crawler, it's required just for development

i would like to do that, but we can see that your tests in `taskcluster` install `requirements.txt`, so it would be inefficient 😿

File: test-requirements.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+importlib-metadata==4.13.0
```
Comment (marco-c) [inline test-requirements.txt:1]: Could you add a comment pointing to this PR and saying that it could be removed in the future?

Comment (nnrepos) [reply test-requirements.txt:1]: sure

**Review by marco-c — APPROVED** [review]

**Comment by marco-c** [comment]
Thanks!

**Pull request merged**